api/turing/cluster/servicebuilder/router.go

```diff
@@ -491,7 +491,13 @@ func buildFiberConfigMap(
 	}
 
 	if ver.Ensembler != nil && ver.Ensembler.Type == models.EnsemblerStandardType {
-		propsMap["experiment_mappings"] = ver.Ensembler.StandardConfig.ExperimentMappings
+		if ver.Ensembler.StandardConfig.ExperimentMappings != nil &&
+			len(ver.Ensembler.StandardConfig.ExperimentMappings) != 0 {
+			propsMap["experiment_mappings"] = ver.Ensembler.StandardConfig.ExperimentMappings
+		}
+		if ver.Ensembler.StandardConfig.RouteNamePath != "" {
+			propsMap["route_name_path"] = ver.Ensembler.StandardConfig.RouteNamePath
+		}
```

Comment:
These conditional values ensure that only non-nil/non-empty values are passed to the fiber config.